source/04-PR/04.ptx

```diff
@@ -10,22 +10,185 @@
     </ul>
     </objectives>
     <subsection>
-       <theorem>
+      <title>Activities</title>
+      <activity xml:id="Quadratic-Zeros">
+        <introduction>
+          <remark xml:id="x-intercepts">
+           
+    <statement><p> Recall that to find the <m>x</m> intercepts of a linear or quadratic function, let <m>y=0</m> and solve for <m>x</m>.</p>
+    </statement></remark></introduction>
+
+        <task> <p>What is the factored form of the function, <m>f(x)=x^2+2x-15</m>? 
+         <ol marker= "A."><li> 0</li>
+          <li> <m>(x-3)(x+5)</m></li>
+          <li> <m>(x+3)(x-5)</m></li>
+          <li> <m>(x-3)(x-5)</m></li>
+          <li> <m>(x+3)(x+5)</m></li>
+          </ol>
+        </p>
+        </task>
+        <task> <p>What are the <m>x</m> intercepts of the function above? 
+          <ol marker= "A."><li> 0</li>
+            <li> <m>-3</m> and <m>5</m></li>
+            <li> <m>3</m> and <m>-5</m></li>
+            <li> <m>-3</m> and <m>-5</m></li>
+            <li> <m>3</m> and <m>5</m></li>
+            </ol>
+        </p>
+        </task>
+        <task><p> How do the <m>x</m> intercepts relate to the factored form of the function?
+        </p></task>
+      
+      </activity> 
+      <definition xml:id="Real-zeros"> 
+        <statement> <p>Real zeros of a polynomial function are the same as the <m>x</m> intercepts.
+        </p> 
+      </statement>
+    </definition>
+      
+      <theorem>
         <title>Factor Theorem</title> 
         <statement> 
-          <p> A number c is a zero of a polynomial function f(x) if and only if 
-             x-c  is a factor of f(x).</p>
+          <p> A number <m>c</m> is a zero of a polynomial function <m>f(x)</m> if and only if 
+             <m>x-c</m>  is a factor of <m>f(x)</m>.</p>
         </statement>
       </theorem>
+
+      <activity xml:id="zeros-cubic-factoring">
+        <introduction>
+          <task><p>Find the zeros of the polynomial by factoring <m>f(x)=x^3-x^2-56x</m>.
```

Comment:
Your `<task>`s should follow an `<introduction>`, not be nested inside.  For something like this with only one real task, you can instead just use a `<statement>` without an `<introduction>`.